build/index.php

```diff
@@ -260,7 +260,7 @@
 
 .printable { padding: 2rem; }
 
-h1 { text-align: center; }
+h1, h4 { text-align: center; }
```

Comment:
I added a subheader on the preview edits page that tells the user they need to scroll down to continue editing or save the page